src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -139,5 +197,6 @@ public function reset()
     {
         $this->lists = [];
         $this->views = [];
+        Cache\AbstractStaticOption::reset();
```

Comment:
why? if this is static, it shouldn't need to be reset between requests.

Reply:
It's consistent with the lists and the views, the options used to build them may be defined per request